badger/cmd/write_bench.go

```diff
@@ -247,16 +263,66 @@ func writeBench(cmd *cobra.Command, args []string) error {
 	return err
 }
 
-func reportStats(c *y.Closer) {
+func reportStats(db *badger.DB, c *y.Closer) {
 	defer c.Done()
 
 	t := time.NewTicker(time.Second)
 	defer t.Stop()
+
 	for {
 		select {
 		case <-c.HasBeenClosed():
 			return
 		case <-t.C:
+			txn := db.NewTransaction(false)
+			defer txn.Discard()
+
+			iopt := badger.DefaultIteratorOptions
+			iopt.AllVersions = true
+			iopt.InternalAccess = true
+
+			it := txn.NewIterator(iopt)
+			defer it.Close()
+			for it.Rewind(); it.Valid(); it.Next() {
+				i := it.Item()
+				if bytes.HasPrefix(i.Key(), []byte("!badger!")) {
+					internalKeyCount++
+				}
+				if bytes.HasPrefix(i.Key(), []byte("!badger!Move")) {
+					moveKeyCount++
+				}
+				if i.IsDeletedOrExpired() {
+					invalidKeyCount++
+				} else {
+					validKeyCount++
+				}
+			}
+
+			// fetch directory contents
+			if showDir {
+				fmt.Printf("The directory contents are:\n")
+				err := filepath.Walk(sstDir, func(path string, info os.FileInfo, err error) error {
+					fileSize := strconv.FormatFloat(float64(info.Size()) / (1024.0 * 1024.0),
+						'f', 2, 64)
+					files = append(files, path + " " + fileSize + " MB")
+					if filepath.Ext(path) == ".vlog" {
+						vlogCount++
+					}
+					if filepath.Ext(path) == ".sst" {
+						sstCount++
+					}
+					return nil
+				})
+				if err != nil {
+					log.Printf("Error while fetching directory. %v.", err)
+				} else {
+					for _, file := range files {
```

Comment:
```suggestion
                                        fmt.Printf("[Content] Number of files:\n", len(files))
					for _, file := range files {
```
Something like this?
Keeping a count would be good for later processing (if automated).